skorch/callbacks/lr_scheduler.py

```diff
@@ -34,13 +28,19 @@ class LRScheduler(Callback):
     policy : str or _LRScheduler class (default='WarmRestartLR')
       Learning rate policy name or scheduler to be used.
 
+    monitor : str or callable (default=None)
+      Value of the history to monitor or callback that determines whether
+      this epoch should to a checkpoint. The callback takes the network
+      instance as parameter.
```

Comment:
Maybe add a note what values to expect (in case of `Checkpoint` we want boolean values, here we want real values, right?)